src/providers/http.ts

```diff
@@ -361,19 +369,21 @@ export class HttpProvider implements ApiProvider {
     }
     try {
       const parsedOutput = (await this.responseParser)(parsedData, rawText);
-      ret.output = parsedOutput.output || parsedOutput;
+      ret.output = parsedOutput?.output ?? parsedOutput;
```

Comment:
this is going to return undefined if parsed output is undefined, is that what we want?

Reply:
probably not, going to revert